apps/nextjs/src/app/api/chat/chatHandler.ts

```diff
@@ -71,6 +74,12 @@ async function setupChatHandler(req: NextRequest) {
         chatId,
       );
 
+      const threatDetectors = [
+        new BasicThreatDetector(),
+        new HeliconeThreatDetector(),
+        new LakeraThreatDetector(),
```

Comment:
For now shall we just use Lakera? I think Helicone one can go, and maybe not use the Basic one either, so we can trial Lakera in isolation

Reply:
No we still need this if the Helicone proxy is on. If it's off then this has no effect

Reply:
Yep, okay. Do we still need the 'Basic' detector running? Feels superfluous